### DISCUSSION TITLE
Charming example for Leptos SSR/Hydrate. Chart Auto Resize/Chart Update example.

### DISCUSSION THREAD

**Discussion by sangelxyz** [yuankunzhang/charming#118] [Show and tell]
This is just something i have put together feel free to copy it. 

https://github.com/sangelxyz/charming-leptos-ssr

It's using the leptos SSR+Hydrate template.
Charming Wasm, basic line chart example

I have implemented the resize feature of charming, i converted the TS resize interface to Rust.

I have also added reactivity by the leptos on-click event to update the chart.

**Comment by LukaOber** [comment]
Looks great, thank you very much for providing a more thorough leptos example. But I think it is a bit too much to add as a standard example in the repo, do you agree?
